official/docs/node/current/carrier-metadata/retrieve.js

```diff
@@ -0,0 +1,16 @@
+const EasyPostBetaClient = require('@easypost/api/src/beta/easypost');
+
+const client = new EasyPostBetaClient(process.env.EASYPOST_API_KEY);
+
+(async () => {
+  // Request all metadata for all carriers
+  const carrierMetadata = await client.CarrierMetadata.retrieveCarrierMetadata();
+  console.log(carrierMetadata);
+
+  // Request specific metadata for specific carriers
+  const carrierMetadataWithFilters = await client.CarrierMetadata.retrieveCarrierMetadata(
```

Comment:
```suggestion
  const carrierMetadata = await client.CarrierMetadata.retrieveCarrierMetadata(
```

Inconsistent variable naming from other language samples

Reply:
Because you can't have multiple constants with the same name. Needs to be slightly different here